pgcli/main.py

```diff
@@ -212,6 +212,24 @@ def register_special_commands(self):
                                 'Send all query results to file.')
         self.pgspecial.register(self.info_connection, '\\conninfo',
                                 '\\conninfo', 'Get connection details')
+        self.pgspecial.register(self.change_table_format, '\\T', '\\T [format]',
+                                'Change the table format used to output results')
+
+    def change_table_format(self, pattern, **_):
+        try:
+            if pattern not in TabularOutputFormatter().supported_formats:
+                raise ValueError()
+            self.table_format = pattern
+            yield (None, None, None,
+                   'Changed table format to {}'.format(pattern))
+        except ValueError:
+            msg = 'Table format {} not recognized. Allowed formats:'.format(
+                pattern)
+            for table_type in TabularOutputFormatter().supported_formats:
+                msg += "\n\t{}".format(table_type)
+            msg += '\ncurrently table format: %s' % self.table_format
```

Comment:
`currently table format` sounds a bit awkward. How about `Current value: XXX` or `Currently set to: XXX`?